Makefile

```diff
@@ -51,6 +51,9 @@ test-db:
 	python -m pytest tests/db
 	python -m pytest tests/queries
 
+test-integration:
+	python -m pytest tests/integration
+
 test-all:
```

Comment:
`test-integration` is not included in `test-all`

Reply:
I don't want it there for now. Eventually we will not need these tests anymore. Once we fully remove all essence of V1.